src/containers/SubjectViewer.jsx

```diff
@@ -0,0 +1,252 @@
+import React, { PropTypes } from 'react';
```

Comment:
This should probably pull `PropTypes` from the `prop-types` package instead of `'react'`.